components/views/SourcecredView.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react'
+import styled from 'styled-components'
+import { H4, H2, Body1 } from '../ui/Typography'
+import Button from '../ui/Button'
+import useCommunity from '../hooks/useCommunity'
+import useWeb3 from '../hooks/useWeb3'
+
+const IntegrationsHeader = styled.div`
+  margin-bottom: 3.6rem;
+`
+
+const IpfsLink = styled.div`
+  a {
+    color: ${props => props.theme.colors.white};
+  }
+`
+
+const Spinner = styled.div`
+  border: 0.4rem solid ${props => props.theme.colors.white};
+  border-radius: 50%;
+  border-left-color: transparent;
+  width: 3.6rem;
+  height: 3.5rem;
+  animation: spin 1s linear infinite;
+
+  & > * {
+    @keyframes spin {
+      0% {
+        transform: rotate(0deg);
+      }
+
+      100% {
+        transform: rotate(360deg);
+      }
+    }
+  }
+`
+
+const SourcecredView = () => {
+  const [membersData, setMembersData] = useState(null)
+  const [loading, setLoading] = useState(false)
+  const { isAdmin, currentCommunity } = useCommunity()
+  const { disconnect } = useWeb3()
+
+  const handleSubmit = async () => {
+    try {
+      setLoading(true)
+      setMembersData(null)
+      const res = await fetch('/api/webhooks/sourcecred', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
we could encapsulate later the API calls to avoid rewriting hears and so and DRY it up.